IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDWebviewUIController.m

```diff
@@ -128,7 +128,7 @@ - (NSWindow *)obtainSignInWindow
     NSRect centerRect = [self getCenterRect:windowRect rect2:NSMakeRect(0, 0, DEFAULT_WINDOW_WIDTH, DEFAULT_WINDOW_HEIGHT)];
     
     NSWindow *window = [[NSWindow alloc] initWithContentRect:centerRect
-                                                   styleMask:NSTitledWindowMask | NSClosableWindowMask
+                                                   styleMask:NSWindowStyleMaskTitled | NSWindowStyleMaskClosable
```

Comment:
we still support 10.11. What's the implication for compiling this on macOS 10.11?

Reply:
It's just an enum case, with the same constant as the old version. As long as you're _compiling_ on 10.12 or later, it will work fine even when that binary is deployed to a device running macOS 10.11.

To verify, I set the deployment target to 10.11 and it built. I'll also push that change as well, since right now there is no deployment target set to ensure it builds on old macOS versions...

My understanding is that ADAL (and thus I assumed MSAL) for Obj-C requires Xcode 10, which means they also require macOS 10.13.6 or later for building. If that's not correct let me know and I'll add some Availability.h conditional compilation flags around this.

-Remy